applications/pusher/doc/pusher.md

```diff
@@ -34,3 +63,30 @@ In order for the push services from Apple / Firebase to work they need to be con
 * `sup pusher_maintenance add_firebase_app(AppId, Secret)`
 * `sup pusher_maintenance add_apple_app(AppId, CertFile)` (uses the default APNs host: api.push.apple.com)
 * `sup pusher_maintenance add_apple_app(AppId, CertFile, Host)` (uses a custom APNs host, i.e. api.development.push.apple.com)
+
+
+1. start pusher and enable pusher role in kamailio
+../kamailio/local.cfg
+
+2. Initialize pusher app.
+
+[root@apps001 ~]# sup kapps_controller start_app pusher
+{ok,[fcm,gcm,base64url,gun,apns,pusher]}
+[root@apps001 ~]# sup kapps_controller running_apps
+[blackhole,callflow,cdr,conference,crossbar,fax,hangups,media_mgr,milliwatt,omnipresence,pivot,pusher,registrar,reorder,stepswitch,sysconf]
+
+pusher doc in system_config will be created.
+
+
+3. Add Apple and google/firebase credentials into pusher doc using sup commands. The Apple cert needs to be in PEM (text, no encryption) format and should be located in the server.
```

Comment:
move this (remove  number)  and combine with what is already mentioned in the 1st paragraph of `maintenance`